From 4b0d2ae793435e43afea004435dc006afb56eca0 Mon Sep 17 00:00:00 2001
From: Matt McCormick (thewtex) <matt@mmmccormick.com>
Date: Tue, 29 Sep 2009 13:56:49 -0500
Subject: [PATCH 2/2] fix assert error syntax in bibstyles/shared.py

fixes
/usr/lib/python2.6/site-packages/bibstyles/shared.py:297: SyntaxWarning: assertion is always true, perhaps remove parentheses?
  assert(isinstance(template,str),"Must provide a name-template string to make a NameFormatter object.")
---
 bibstyles/shared.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/bibstyles/shared.py b/bibstyles/shared.py
index 0ccc14c..e880fb6 100644
--- a/bibstyles/shared.py
+++ b/bibstyles/shared.py
@@ -294,7 +294,7 @@ class NameFormatter(object):
 		sets the name formatting template *and* sets the associated partdict used for actual formatting 
 		"""
 		shared_logger.debug("NameFormatter.set_template args: "+str(template))
-		assert(isinstance(template,str),"Must provide a name-template string to make a NameFormatter object.")
+		assert isinstance(template,str), "Must provide a name-template string to make a NameFormatter object."
 		self._template = template
 		self.partdict = self.template2dict(template)
 	template = property(get_template,set_template,None,"template property")
-- 
1.6.5.rc1

